batch/Makefile

```diff
@@ -95,3 +105,15 @@ test-deploy: push-test
 
 clean:
 	rm -f $(PY_CHECKERS) && rm -rf batch-secrets
+
+clean-db-production-local: batch-secrets/batch-production-cloud-sql-config.json
+	CLOUD_SQL_CONFIG_PATH=batch-secrets/batch-production-cloud-sql-config.json ./clean-db.sh
+
+clean-db-production:
+	CLOUD_SQL_CONFIG_PATH=/batch-secrets/batch-production-cloud-sql-config.json ./clean-db.sh
```

Comment:
Typo? I think batch-secrets/batch-production-cloud-sql-config.json, not from root?